docs/user_guide/celery.md

```diff
@@ -0,0 +1,205 @@
+# Celery With Merlin
+
+Merlin uses [Celery](http://www.celeryproject.org), a Python based distributed task management system. Merlin uses Celery to queue work which is processed by Celery workers.
+
+Merlin queues tasks to the broker which receives and routes tasks. Merlin by default is configured to use [RabbitMQ](https://www.rabbitmq.com/) as the broker but [Redis](https://redis.io/) can be used as well.
+
+Celery has many functions, it defines the interface to the task broker, the backend results database and the workers that will run the tasks.
+
+As discussed in the [Configuration](./configuration/index.md) page, the broker and backend are configured through [the app.yaml file](./configuration/index.md#the-appyaml-file). A configuration for the rabbit AMQP server is shown below.
+
+???+ abstract "Config File for RabbitMQ Broker and Redis Backend"
+
+    ```yaml title="app.yaml"
+    celery:
+        # directory where Merlin looks for the following:
+        # mysql-ca-cert.pem rabbit-client-cert.pem  rabbit-client-key.pem redis.pass
+        certs: /path/to/celery/config
+
+    broker:
+        # can be rabbitmq, redis, rediss, or redis+sock
```

Comment:
We need to add ampqs to the list.

Reply:
BROKERS: List[str] = ["rabbitmq", "redis", "rediss", "redis+socket", "amqps", "amqp"]

Reply:
will do